### ISSUE TITLE
PDF Export Module error

### ISSUE THREAD

**Issue by robertnixon2003** [MISP/misp-modules#172]
Getting this error when trying to use the PDF Export Module:

```
ERROR:misp-modules:Something went wrong:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/misp_modules/__init__.py", line 197, in post
    response = yield tornado.gen.with_timeout(timeout, self.run_request(jsonpayload))
  File "/usr/local/lib/python3.5/dist-packages/tornado/gen.py", line 1099, in run
    value = future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.5/dist-packages/misp_modules/__init__.py", line 185, in run_request
    response = mhandlers[x['module']].handler(q=jsonpayload)
  File "/usr/local/lib/python3.5/dist-packages/misp_modules/modules/export_mod/pdfexport.py", line 150, in handler
    report.asciidoc()
  File "/usr/local/lib/python3.5/dist-packages/misp_modules/modules/export_mod/pdfexport.py", line 133, in asciidoc
    self.report += self.event_level_tags()
TypeError: Can't convert 'NoneType' object to str implicitly
```

**Comment by robertnixon2003** [comment]
Any update on this?

**Comment by SteveClement** [comment]
Can you try the latest version?
I refactored some stuff, if still broken it is a python3.5 thing.
But quite honestly it is not really useful atm.

**Comment by Vincent-CIRCL** [comment]
A bugfix is available with #278. If accepted, this would solve - at least with very basic pdf - the issue.
